OpenRA.Mods.Cnc/UtilityCommands/ImportTSMapCommand.cs

```diff
@@ -115,8 +115,8 @@ class ImportTSMapCommand : IUtilityCommand
 			{ 0xAF, "trock03" },
 			{ 0xB0, "trock04" },
 			{ 0xB1, "trock05" },
-			{ 0xBB, "veinholedummy" },
-			{ 0xBC, "crate" }
+
```

Comment:
I'd say either remove the gap, or add a comment here, too.

Reply:
Okay.